### DISCUSSION TITLE
Required more server instances

### DISCUSSION THREAD

**Discussion by AhmadRazaKhokhar1** [community/community#147315] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Please add more server isntances as I am getting the following error again and again:
<img width="1080" alt="Screenshot 2024-12-17 at 7 45 10 PM" src="https://github.com/user-attachments/assets/5a96c40b-e0d3-4f9c-a360-f4024e2ee886" />

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @AhmadRazaKhokhar1 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by kenskiyyy** [comment]
same for me, the whole website is wayy too slow

**Comment by tembogab** [comment]
same here

**Comment by yarikoptic** [comment]
same here from VT, USA -- can't navigate actions, posting comments too slow, unicorn with "more servers" too

**Comment by SerhiiKorolik** [comment]
+1

**Comment by queenofcorgis** [comment, marked as answer]
Thanks everyone, we appreciate you reaching out to report this issue! 


We have provided details about the incident in [this discussion](https://github.com/orgs/community/discussions/147316). Please subscribe to that post to receive real-time updates and a notification when the incident is resolved.
